src/sentry/api/endpoints/group_tagkey_details.py

```diff
@@ -49,7 +49,7 @@ def get(self, request: Request, group, key) -> Response:
 
         if group_tag_key.count is None:
             group_tag_key.count = tagstore.backend.get_group_tag_value_count(
-                group.project_id, group.id, environment_id, lookup_key, tenant_ids=tenant_ids
+                group, environment_id, lookup_key, tenant_ids=tenant_ids
             )
```

Comment:
this call couldn't have succeeded as written -- I don't see any errors in prod so perhaps this codepath is dead?